### ISSUE TITLE
Galatic War

### ISSUE THREAD

**Issue by TwinNumberOne** [siggame/MegaMinerAI-Game-Ideas#72]
**TLDR**
-
- Two players at the opposite ends of a mirrored galaxy
- Players can only send unites along fixed paths to different nodes.
- At the galactic core there is a ring of arbitrarium that is the victory points of the game
- if possible make each system have a special effect per type of star or if it has a black whole

**Story**
-
Two different unified planets discover hyper lanes used by an extinct ancient civilization and also instant communication, at the same time. So what do they do with this information? Go to war of course! They are fighting for the right to control the entire galaxy. 

**The Game**
-
**Victory**
Destroy all the enemy empires homeworld or mine the most arbitrarium by the end of the time limit of the game

**Map**
- Both players start in the game on their homeworld which has their homeworld (produces a medium amount of each material). There will be a variety of systems each that has a focus of fuel or materials or a balance with some systems having nothing due to just being a star or a black hole. But the big thing about the map is that each system has a limited amount of resources, and that to gain resources your ships must travel back to the homework to drop them off.

**Movement**
Each ship will have a system move speed that says how fast they should be able to jump node to node.

**Resource Collection**
There are two none victory resources in the game Fuel, and ore to be collected they need to be brought to the homeworld or a shipyard.

**Unites**
The Frigate, Destroyer and Battleship are meant to be the core of any fleet with the carrier meant to be the back up with its ability to produce/launch a fighter every turn. The defensive units are the Titan as it moves so slow it is hard to use it to press the attack but it does ok damage and the minefield which either slows the enemy down due to the time to clear it or by instantly killing a certain amount of ships by them ignoring it and moving through and it can be built at the homeworld/shipyard and moved around via transport ships. Then there is the resource collection unites the Miner, and the Transport I bet you can guess what they do. The shipyard acts as a forward base but it's vulnerable to being counter-attacked due to its lack of defenses and slow speed.
Cost(ore) the prices/stats of the ships will range from 1-5 (ranking)
Combat order: Fridgate -> Destroyer -> Battleship -> Titan -> Strike Craft -> Carrier -> Shipyard -> Transport
The carrier, shipyard, and transport should be defended from damage from their own team's ships, also maybe make the Titan draw "agro" via forcing it to be taken out first?

**Frigate (Strong sv Destroyer,  Weak vs battleship)**
The lowest cost combat unite meant to be the most spammed, shortest range
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|2|2|2|4|2|2|

**Destroyer (Strong vs Battleship,  Weak vs Frigate)**
The more medim class ship meant to cover from the frigates from the battleship, medium range
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|3|3|3|3|3|3|

**Battleship (Strong vs Frigate, Weak vs Destroyer)**
Ment to clear to destroyers so make sure that the frigate so the destroyers can take out the battleships, longest range
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|4|4|4|2|4|4|

**Carrier (Neutral to everything with its fighters, fighter squads take 3 shots to destroy)**
nearly a mobile projection and carrying platform for fighters little personal defenses. Is attacked last but before the shipyard
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|4|4|1|2|4|2|

**Transport (Moves all resources)**
basically defenseless but it moves fast and carrys allot
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|2|2|1|4|5|2|

**Miner (Mines all resources)**
basically defenseless but it moves at a decent speed and carrys a good amount of stuff
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|2|3|1|3|3|2|

**Shipyard (Slow movement produces units)**
Draws from empire storage to build ships.
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|5|5|1|1|5|1|

**Titan (Slow, medium dmg)**
Ment to defense it has a long range but not too high of dmg.
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|3|4|2|1|2|1|

**Strike Craft (Carrier made unit)**
Neutral damage to all
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|1|1|3|5|1|1|

**minefield**
Can be attacked to be removed or do damage to a unite if it moves through its node (enemy only)
| **Cost** | **HP** | **DMG** | **Movement** | **Storage** | **Fuel Cost** | 
|----|----|----|----|----|----|
|2|5|5|0|0|0|

**Combat**
The purpose of this combat is to be built around an advantage system where certain ships do bonus damage to other ships with a more wild card in the carrier to make sure to keep it fresh along with mines to slow down or kill attackers. It should be noted that minefields can be carried by transporters. With the unites move speeds and ranges it makes it so that your approach becomes just as important as unite composition allowing for a relatively simplistic but rewarding strategy. Also, fighters can only be carried by carriers of which carriers can only do one thing per turn launch one set of fighters or built a set of fighters.

**Remarks**
I hope the whole hyper lane idea is not too hard to do the point of it is to introduced chokepoint systems and to allow combat to happen and by limiting resources in the individual systems we can push conflict even more as they will need more to fuel their empire.
Maybe allow players to build worm whole generators that they need to deploy (one way?) they can be very slow-moving to allow better troop deployment but I fear the creep of it.
